go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	k8s.io/component-base v0.27.3
 	k8s.io/kubectl v0.27.3
 	k8s.io/kubelet v0.27.3
-	k8s.io/kubernetes v1.27.3
+	k8s.io/kubernetes v1.15.0-alpha.0
```

Comment:
This one looks like it'll be fixed by https://github.com/openshift/machine-config-operator/pull/3913